images/router/haproxy/conf/haproxy-config.template

```diff
@@ -254,6 +254,7 @@ backend be_secure_{{$cfgIdx}}
 {{     range $idx, $cfg := $serviceUnit.ServiceAliasConfigs }}
 {{       if and (ne $cfg.Host "") (eq $cfg.TLSTermination "")}}
 {{$cfg.Host}}{{$cfg.Path}} {{$idx}}
+{{$cfg.Host}}{{env "ROUTER_SERVICE_HTTP_PORT" "80"}}{{$cfg.Path}} {{$idx}}
```

Comment:
Missing  separator between host and port. `{{$cfg.Host}}:{{env "ROUTER_SERVICE_HTTP_PORT" "80"}}{{$cfg.Path}}`.